### ISSUE TITLE
18 Digit Requirement for Server IDs

### ISSUE THREAD

**Issue by OfficialCRUGG** [chookscord/framework#93]
When creating a new project, I am required to enter a Dev Server ID. For whatever reason, there is a requirement that it has to be exactly 18 digits. Why is that?

**Comment by 0xDevansh** [comment]
All Snowflake ID's in Discord are 18 digits long, be it a server, user or channel ID. You can get your server ID by right clicking on the server icon in the sidebar. ([Enable developer mode first](https://www.howtogeek.com/714348/how-to-enable-or-disable-developer-mode-on-discord/))

**Comment by OfficialCRUGG** [comment]
That's not true. Snowflakes are based on the timestamp and therefore can go longer with time. If you create a new server or account today, its ID will be 19 characters long. At some point in the future, it will be 20, and so on.

**Comment by 0xDevansh** [comment]
Interesting, I'm sorry I was misinformed.
